boms/src/main/resources/bom-template.xml

```diff
@@ -60,11 +59,6 @@
         <artifactId>spi</artifactId>
         <version>${project.version}</version>
       </dependency>
-      <dependency>
-        <groupId>org.wildfly.swarm</groupId>
-        <artifactId>config-api-runtime</artifactId>
-        <version>${version.wildfly-config-api-runtime}</version>
-      </dependency>
```

Comment:
Whoops!

Reply:
Oh, I thought I fixed it, I was testing your fix and it made it into the jose branch by accident and somehow it ended up deleted in the end, will put it back